packages/cli/src/cmds/assert/types.d.ts

```diff
@@ -0,0 +1 @@
+export type Scope = 'appmap' | 'http_server_response' | 'http_server_request';
```

Comment:
The `response` is always paired with the `request` so I think only `appmap` and `http_server_request` are needed here.